examples/opencv_rc.py

```diff
@@ -0,0 +1,246 @@
+    blurred_img = cv.GaussianBlur(resized_img, (3, 3), 0)
+    sharp_img = cv.addWeighted(resized_img, 1 + SHARP_AMOUNT, blurred_img,
+                               -SHARP_AMOUNT, gamma=SHARP_GAMMA)
+
+    # Send the image back to the main thread for display
+    IMG_QUEUE.put(sharp_img)
+
+
+def stop_all(cli, state):
+    """
+    This function simply stops all motors and resets the corresponding
+    velocities. It is used when Cozmo detects that it has been picked up, or is
+    about to fall off a cliff.
+    :param cli: An instance of the pycozmo.Client class representing the robot.
+    :param state: A boolean set to True if Cozmo has been picked up or is about
+    to fall off a cliff. False otherwise.
+    :return: None
+    """
+    global LIN_VELOCITY, ANG_VELOCITY
+
+    # Well as said above, if Cozmo is not touching the ground anymore we stop
+    # all motors to prevent any damage
+    if state:
+        # Stop the motors
+        cli.stop_all_motors()
+
+        # Reset the linear and angular velocities
+        LIN_VELOCITY = 0
+        ANG_VELOCITY = 0
+
+
+if __name__ == "__main__":
+    # Connect to the robot
+    with pc.connect() as cli:
+        try:
+            # Look forward
+            cli.set_head_angle(HEAD_TILT)
+
+            # Enable the camera
+            cli.enable_camera(enable=True, color=COLOR_IMG)
+
+            # Set the lift in its minimum position
+            cli.set_lift_height(height=LIFT_HEIGHT)
+
+            # Handle new incoming images
+            cli.add_handler(pc.event.EvtNewRawCameraImage, on_camera_img)
+
+            # Handle cliff and pick-up detection
+            cli.add_handler(pc.event.EvtCliffDetectedChange, stop_all)
+            cli.add_handler(pc.event.EvtRobotPickedUpChange, stop_all)
+
+            # Loop forever
+            while True:
+                try:
+                    # Get the next frame from the camera
+                    # A timeout is applied so that the robot might still be
+                    # controlled even if no image can be displayed
+                    img = IMG_QUEUE.get(timeout=0.2)
+                    # Display the frame in a window
+                    cv.imshow('Camera', img)
+                    IMG_QUEUE.task_done()
+                except Empty:
+                    logging.warning('Did not get any image from the camera so '
+                                    'not displaying any.')
+
+                # Read the next key event
+                # /!\ It should be noted that that if OpenCV's window displaying
+                # the image received from the camera loses focus, then Cozmo
+                # will not answer your commands anymore.
+                key = cv.waitKeyEx(1)
+
+                # Act accordingly
+                if key == ord('q'):
+                    # Exit the program
+                    break
+
+                # Losing a bit of computational time to prevent sending motor
```

Comment:
I don't think you are missing any computational time anywhere, just implementing a feature!

(I just wanted to say that there is no loss here that I can see at least, just fine code ;))